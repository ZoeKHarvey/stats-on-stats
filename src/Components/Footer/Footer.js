
import { getTeams, isLoading, hasError } from '../../actions';
import { bindActionCreators } from 'redux';
import { fetchTeams } from '../../apiCalls';
import { connect } from 'react-redux';
import React, { Component } from 'react';
import TeamContainer from '../TeamContainer/TeamContainer';
import { Link } from 'react-router-dom';
import fullrink from '../../images/full-rink.jpeg'
import './Footer.scss'
import ice from '../../images/ice2.jpg';
import NavLogos from '../NavLogos/NavLogos'


export const  Footer = () => {

    return(
      <section className="section-footer">
        <div className="footer__div--links">
<Link to='/teams'className="footer__a--link"> 
  <h3>Teams</h3>
</Link>
<Link to='/standings' className="footer__a--link">
  <h3>Standings</h3>
</Link>
</div>
      </section>
    )
  }

  export default Footer