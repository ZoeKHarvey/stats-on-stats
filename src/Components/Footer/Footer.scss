.footer__div--links {
  display: flex;
  justify-content: space-around;
  background: rgba(55,63,81,.4);
  text-decoration: none;
  color: black;
}

.footer__a--link {
  text-decoration: none;
  color: black;
  font-family: 'Barlow Condensed', sans-serif;
  font-size: 1.5em;
}

h3:hover {
  transform: scale(1.4)
}

