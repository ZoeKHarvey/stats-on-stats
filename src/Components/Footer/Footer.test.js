import React from 'react';
import { shallow } from 'enzyme';
import { Footer } from './Footer';

describe('Footer', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(
      <Footer />
    );
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  })
})
