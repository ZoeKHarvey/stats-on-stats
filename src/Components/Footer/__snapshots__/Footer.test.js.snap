// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match the snapshot 1`] = `
<section
  className="section-footer"
>
  <div
    className="footer__div--links"
  >
    <Link
      className="footer__a--link"
      to="/"
    >
      <h3>
        Home
      </h3>
    </Link>
    <Link
      className="footer__a--link"
      to="/teams"
    >
      <h3>
        All Teams
      </h3>
    </Link>
    <Link
      className="footer__a--link"
      to="/favorites"
    >
      <h3>
        Favorite Players
      </h3>
    </Link>
  </div>
</section>
`;
